source/install/cloud-changelog.md

```diff
@@ -3,12 +3,62 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2022-01-19](#release-2022-01-19)
 - [Release 2022-01-11](#release-2022-01-11)
 - [Release 2021-12-08](#release-2021-12-08)
 - [Release 2021-12-01](#release-2021-12-01)
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
-- [Release 2021-11-10](#release-2021-11-10)
+
+## Release 2022-01-19
+
+### Improvements
+
+#### User Interface (UI)
```

Comment:
- Invite to team modal now auto-focuses its email search input.
 - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
 - Updated **Account Settings** terminology to **Settings**.
 - Added Accept-Language header to generate link previews in the Default Server language.
 - The "Invite Members" button is now hidden when the Direct Message category is collapsed.

Reply:
Removed items that do not impact Cloud